components/SearchBar.vue

```diff
@@ -2,92 +2,180 @@
+                    data-testid="search-bar-language"
+                >
+                    <option
+                        value=""
+                        class="text-primary-text-muted hidden"
+                        disabled
+                        selected
+                    >
+                        {{ $t("searchBar.selectLanguage") }}
                     </option>
-                    <option :key="language.code" :value="language.code" v-for="(language, index) in languageDropdownOptions">
+                    <option
+                        :key="language.code"
+                        :value="language.code"
+                        v-for="(language, index) in languageDropdownOptions"
+                    >
                         {{ language.simpleText }}
                     </option>
                 </select>
             </div>
         </div>
         <div id="search-button" class="flex items-center">
-            <button id="searchButton" class="flex flex-0 flex-row rounded-full bg-primary w-28 pl-1 pr-2 py-2 text-sm align-middle justify-center
-                    hover:bg-primary-hover transition-all" @click="search" data-testid="search-button">
-                <svg role="img" alt="search icon" title="search icon"
-                    class="search-icon w-5 h-5 mr-1 fill-primary-text-inverted">
-                    <use xlink:href="../assets/images/search-icon.svg#search-icon-svg" />
+            <button
+                id="searchButton"
+                class="flex flex-0 flex-row rounded-full bg-primary w-28 pl-1 pr-2 py-2 text-sm align-middle justify-center hover:bg-primary-hover transition-all"
+                @click="search"
+                data-testid="search-button"
+            >
+                <svg
+                    role="img"
+                    alt="search icon"
+                    title="search icon"
+                    class="search-icon w-5 h-5 mr-1 fill-primary-text-inverted"
+                >
+                    <use
+                        xlink:href="../assets/images/search-icon.svg#search-icon-svg"
+                    />
                 </svg>
-                <span class="self-center text-primary-text-inverted">{{ $t('searchBar.search') }}</span>
+                <span class="self-center text-primary-text-inverted">{{
+                    $t("searchBar.search")
+                }}</span>
             </button>
         </div>
     </div>
 </template>
 
 <script setup lang="ts">
-import { ref, Ref, watchEffect } from 'vue'
-import { useSearchResultsStore } from '~/stores/searchResultsStore.js'
-import { useLocationsStore } from '~/stores/locationsStore.js'
-import { useSpecialtiesStore, SpecialtyDisplayOption } from '~/stores/specialtiesStore.js'
-import { Locale, Specialty } from "~/typedefs/gqlTypes.js"
-import { useLocaleStore, LocaleDisplay } from '~/stores/localeStore.js'
+import { ref, Ref, watchEffect } from "vue";
```

Comment:
We gotta fix this linting issue. We don't want to litter semicolons everywhere. 
I haven't seen it on other people's PRs yet, let's see if someone else's is also doing this or if it's just your machine. 🤔

Reply:
Turned off prettier and it seemed to not change and add all the semicolons! So hopefully I won't keep making this an issue. Sorry again